src/about.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Make the page responsive by adding in custom width using media queries.

Reply:
Media query looking at the device width should be considered with a minimum of 800px